components/camel-jira/src/main/java/org/apache/camel/component/jira/consumer/NewCommentsConsumer.java

```diff
@@ -43,15 +43,17 @@
 
     public NewCommentsConsumer(JiraEndpoint endpoint, Processor processor) {
         super(endpoint, processor);
-        LOG.info("JIRA NewCommentsConsumer: Indexing current issue comments...");
-        getComments(false);
+        LOG.info("JIRA NewCommentsConsumer: Indexing current issue comments.");
```

Comment:
We should move such tasks to doStart and not in constructors

Reply:
Done. Thanks for the review.